apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -0,0 +1,13 @@
+ack_generate_info:
+  build_date: "2023-12-21T21:47:08Z"
+  build_hash: a4bd77b1cacbbbfa7aec7325e5e93dd92d64394b
+  go_version: go1.20.4
+  version: v0.26.0-4-ga4bd77b
```

Comment:
Can we generate the controller using `code-generator` v0.28.0?

Reply:
re-generated using `code-generator` v0.28.0